src/utils/certificates_generator.py

```diff
@@ -8,10 +8,20 @@
 from src.utils.api_helper import ApiHelper
 from src.utils.db_postgress_helper import DbConnect
 
+fake = Faker('en_CA')
+C_NAMES = [fake.company() for x in range(0, 5)]
```

Comment:
remove from this module. we discussed this

Reply:
fixed